### DISCUSSION TITLE
Query encoding fails with certain special chars.

### DISCUSSION THREAD

**Discussion by rcs10** [ghiscoding/Angular-Slickgrid#895] [Ideas]
Im using Angular slickgrid with ODataQuery Backend service api and I have noticed that if the column filter contains certain special chars like ">" ,"<", "*", "!", "=" the encoding fails and from my discussion on [stackoverflow](https://stackoverflow.com/questions/71953422/angular-slickgrid-encode-special-characters-while-searching) , I understand that these chars play special role in formulating the final query.

Any workaround for this issue?  Thanks in advance!!

**Comment by ghiscoding** [comment]
Like I wrote in SO, these chars are special chars and are extracted and removed from the final query if they are detected at the beginning of the Filter string (or also at the end for `*` only). There's no way around it and that is intended. For example if we type `>ba` in the Example 5, it will create a filter that is greater than "ba" `$filter=(Name gt 'ba')` and since it was converted then there is no reason to keep the special char anymore. As for the `#` symbol that you asked in SO, well there's just no use for that symbol and so it is not extracted but kept and it has to be encoded. 

So in short, special chars that can impact the query and are written as the first 1 and/or 2 characters (<, <=, >, >=, <>, !=, *) will be extracted and impact the final OData query and they will also be removed from the query because of the conversion. There's no way around it and it was really intended to be this way.
